NER_Data_collection_and_annotation.ipynb

```diff
@@ -0,0 +1,981 @@
+{
```

Comment:
<p>can you help me understand what this is?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/meetnaren/ner-project/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>

Reply:
<p>Names in the whole piece of text are mistyped in several places. For example, within a single piece of text, the name "John Doe" is spelt as is in one place, and "John Dough" in another. I'm fixing these typos here (before running them through spacy) so that when we augment the dataset by replacing these names, they get replaced in all the occurrences.</p>